geth/common/profiling.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"runtime/pprof"
+)
+
+const (
+	CpuProfilingFilename = "status_cpu.prof"
```

Comment:
If a "subname acronym" starts from an uppercase letter, the convention is to write it fully uppercase: `CPUProfilingFilename` :)